### PR TITLE
chore: deprecate Node.js Security WG DB strategy

### LINKED ISSUE #46: Update documentation to deprecate Node.js Security WG DB
The Node.js Ecosystem Security Database is no more maintained (**it's deprecated**). So there is no reason to use it anymore for a production application. We should tag as deprecated in README (and also in the strategy markdown page).

image

- Throw a deprecation warning when we use this strategy (see [process.emitWarning](https://nodejs.org/api/process.html#processemitwarningwarning-options))

https://github.com/NodeSecure/vulnera/blob/51bb814087da39e105f37a1538c43ca7b430b09b/src/strategies/index.js#L15-L16

Here a link to an official Node.js article that explain why: https://nodejs.medium.com/node-js-ecosystem-vulnerability-reporting-program-winding-down-591d9a8cd2c7

### PR DESCRIPTION
**Pull request by PierreDemailly**
fixes #46

### CODE REVIEW

File: docs/node_security_wg.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Node.js strategy
 
+# ⚠️ Node.js Security WG DB is deprecated. [Here is why](nodejs.medium.com/node-js-ecosystem-vulnerability-reporting-program-winding-down-591d9a8cd2c7).
```
Comment (fraxken) [inline docs/node_security_wg.md:3]: I would prefer to use newest Github markdown feature `> **Warning**: example`

> **Warning**: example

File: src/strategies/security-wg.js [hunk 2]
```diff
@@ -13,6 +14,11 @@ import * as cache from "../cache.js";
 import { standardizeVulnsPayload } from "./vuln-payload/standardize.js";
 
 export async function SecurityWGStrategy(options = {}) {
+  emitWarning('Node.js Security WG DB is deprecated and will be removed soon.', {
```
Comment (fraxken) [inline src/strategies/security-wg.js:17]: It would prefer if you use `process.emitWarning` in one shot (no need to destruct it at the top).

**Review by fraxken — CHANGES_REQUESTED** [review]

File: docs/node_security_wg.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Node.js strategy
```
Comment (fraxken) [inline docs/node_security_wg.md:1]: You can add `(deprecated)` in the main title here

**Review by fraxken — APPROVED** [review]

**Review by antoine-coulon — APPROVED** [review]
LGTM

**Pull request merged**